code/modules/hivemind/mobs.dm

```diff
@@ -546,15 +552,15 @@
 	icon = 'icons/mob/hivemind.dmi'
 	icon_state = "mechiver-closed"
 	icon_dead = "mechiver-dead"
-	health = 450
-	maxHealth = 450
-	melee_damage_lower = 10
-	melee_damage_upper = 15
+	health = 1000
+	maxHealth = 1000 //utter beast of a machine, should be able to hold ground on its own for a while.
+	melee_damage_lower = 30 //10 dmg difference as too reduce its strength, even if by a bit
+	melee_damage_upper = 40 //It crushes you, your bones will probably break
 	mob_size = MOB_LARGE
-	attacktext = "crushes"
+	attacktext = "crushed"
 	ability_cooldown = 1 MINUTES
-	speak_chance = 5
-	speed = 16
+	speak_chance = 8
+	speed = 18
```

Comment:
With this, damage, and rarity buffs this could mean instant death for an unarmed individual that just found it by accident, could use a longer agro time similar to keiser roach.

Reply:
Reduced the mechiver speed and damage. If by agro time you mean how long it takes for the mob to start moving towards you to attack you, they are exactly the same (suprisingly)